db-setup.sql

```diff
@@ -8,29 +8,29 @@ CREATE TABLE actors
 -- keeps track of the play scripts available to read
 CREATE TABLE scripts
 (
-    id       SERIAL PRIMARY KEY,
-    title    VARCHAR NOT NULL,
+    id    SERIAL PRIMARY KEY,
```

Comment:
This commit appears to undo changes made in my recent PR which seems unintentional